rust/actyx/ax/src/certs/developer_certificate.rs

```diff
@@ -70,21 +70,17 @@ impl ManifestDeveloperCertificate {
     }
 }
 
-fn serialize_dev_private_key<S: Serializer>(x: &PrivateKey, s: S) -> Result<S::Ok, S::Error> {
-    s.serialize_str(&x.to_string())
+fn deserialize_dev_private_key<'de, D: Deserializer<'de>>(d: D) -> Result<Option<PrivateKey>, D::Error> {
+    let s = <Option<String>>::deserialize(d)?;
+    s.map(|s| s.parse::<PrivateKey>().map_err(serde::de::Error::custom))
+        .transpose()
 }
 
-fn deserialize_dev_private_key<'de, D: Deserializer<'de>>(d: D) -> Result<PrivateKey, D::Error> {
-    let s = <String>::deserialize(d)?;
-    s.parse::<PrivateKey>().map_err(serde::de::Error::custom)
-}
-
-#[derive(Debug, Clone, Deserialize, Serialize, PartialEq, Eq)]
+#[derive(Debug, Clone, Deserialize, PartialEq, Eq)]
```

Comment:
I seem to have missed this: why serialize is removed?

Reply:
+1 for the comment

Reply:
@rkuhn